A New Ship Zealot of Balaam Capital Ship: The MIL-Claymore

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
i have added the MIL-Claymore, a zealot of balaam capital ship. 

It's a capital ship that's focused on giving a full frontal assault, with heavy plastitanium armor at the front, as well as all of it's weapons located solely on the front, much to the detriment of the rest of the ship. 

The ship gets weaker the further away you get to the front, with the back having exposed thrusters and only having 1 layer of reinforced walls. This fact is worsened by the zealots viewing shield generators and FTL drives as sin, so the Claymore relies on it's armor to keep it alive, due to the absences of the latter. 

The interior of the ship is minimal at best, featuring a radar console, a material reclaimer and a mercenary techfab. Barebones really. 

This ship is designed to be a combat frontliner first and foremost, much to the detriment of other areas. 

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

This'll add a MIL class ship for the mercs to use that's fully focused on charging into combat head-on, and while it is extremely strong at the front, it still lacks a shield generator and an FTL drive, limiting it's combat capacity when attacked at any other front. 

It shall only be available on expedition shipyard consoles, with only mercs having access to the ship. 

## How to test
<!-- Describe the way it can be tested -->

load the .yml in IDE, via loadgrid. Should work out fine. 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

<img width="495" height="872" alt="Claymore Live" src="https://github.com/user-attachments/assets/f8e9daf0-fdbe-4166-bf8b-d9b678653ab6" />
<img width="485" height="879" alt="Claymore" src="https://github.com/user-attachments/assets/e9eb78d8-a1d1-4eba-88be-03f821137f9d" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
- [x] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: added the MIL-Claymore

Rubeebeebee committed:
I want to add another gas mining ship- so here is the HES-Phaeron, a tanker ship with a huge tank, but you'll have to carry either one filtered gas, or unfiltered gas.

Rubeebeebee committed:
i added decals to the twilight

Rubeebeebee committed:
Adding the MIL-Claymore, a zealots of balaam capital ship

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Claymore-branch

Cojoke-dot commented:
I think it needs more forward thrusters. It has wayyyyy too many reverse thrusters too

Rubeebeebee commented:
> I think it needs more forward thrusters. It has wayyyyy too many reverse thrusters too

if anything, its the opposite! But the low amount of reverse thrusters is intentional, so it adds to the "FUCK IT WE BALL" vibe of the full frontal assault spirit of this ship

Rubeebeebee committed:
changed it so the wiring at the front part is more robust (since theoretically that'd be the part that would be damaged the most)

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Claymore-branch

Rubeebeebee commented:
synched the branch! should be updated now!

sleepyyapril commented:
I really don't like the shape of this shuttle, nor should more civ ships have the guns you're giving them.

EctoplasmIsGood commented:
> I really don't like the shape of this shuttle, nor should more civ ships have the guns you're giving them.

I think the shape of the ship is actually really cool, and mercs desperately need stronger ships (however, this is needlessly locked to zealots of balaam so its kinda ehhhh)

Rubeebeebee commented:
> > I really don't like the shape of this shuttle, nor should more civ ships have the guns you're giving them.
> 
> I think the shape of the ship is actually really cool, and mercs desperately need stronger ships (however, this is needlessly locked to zealots of balaam so its kinda ehhhh)

its not locked to the zealots of balaam! sure, its made with the zealots in mind, but any merc willing to pilot a ship with a strong frontal assault loadout with no FTL and no shield is welcome to use it!

any merc can just buy it from the exped shipyard

Rubeebeebee commented:
> I really don't like the shape of this shuttle, nor should more civ ships have the guns you're giving them

thankfully its not a CIV ship, its a MIL ship, meaning its merc locked, so regular civs cant buy it off the bat

Rubeebeebee committed:
fixed mapschema (?)
Removed:  mapInit: true
    paused: true

ScyronX commented:
> > I really don't like the shape of this shuttle, nor should more civ ships have the guns you're giving them.
> 
> I think the shape of the ship is actually really cool, and mercs desperately need stronger ships (however, this is needlessly locked to zealots of balaam so its kinda ehhhh)

I agree with Ectoplasm here, this could very well be the only viable ship accesible to Mercenaries. It is best to make it a mercenary ship and its not like Zealots of Balaam cant use it either it will be naturally accesible to them.

Redrover1760 commented:
The main issue with the ship conceptually is existing balance and the lack of other, high value military ships that aren't locked behind something like finding LP Bravo

The biggest capital military ship is currently the Flashpoint. Upending the flashpoint massively by being much much better as that will be bad for the game.

Avalon-Proto commented:
I think that's the idea of the PR, the Zealots *make* these shuttles. But anyone can buy them. Essentially think of it as the Zealots buying and reusing other ships to make their crazy technicals etc

Rubeebeebee commented:
> > > I really don't like the shape of this shuttle, nor should more civ ships have the guns you're giving them.
> > 
> > 
> > I think the shape of the ship is actually really cool, and mercs desperately need stronger ships (however, this is needlessly locked to zealots of balaam so its kinda ehhhh)
> 
> I agree with Ectoplasm here, this could very well be the only viable ship accesible to Mercenaries. It is best to make it a mercenary ship and its not like Zealots of Balaam cant use it either it will be naturally accesible to them.

heres the thing, it is accessible and purchasable by Mercenaries! It's just made by the zealots, like @Avalon-Proto said!

Rubeebeebee commented:
> The main issue with the ship conceptually is existing balance and the lack of other, high value military ships that aren't locked behind something like finding LP Bravo
> 
> The biggest capital military ship is currently the Flashpoint. Upending the flashpoint massively by being much much better as that will be bad for the game.

I will make more high value military ships. This is a promise

Redrover1760 commented:
I need to set a standard at some point for military ships so this can become a proper thing, old mil ships need a lot of love too

Rubeebeebee committed:
replaced missles with 90mm and 57mm

replaced the bridge windows with plasmaglass

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
<img width="1052" height="741" alt="claymore new wepa" src="https://github.com/user-attachments/assets/86ae8bb7-9c52-4b69-b3d5-628c20779f7f" />
<img width="691" height="615" alt="SUPER FUN" src="https://github.com/user-attachments/assets/10224caa-833a-423f-91e4-906c40a9ddf5" />

replaced the missiles with the 90mm autocannon and the 57mm mortar

Rubeebeebee committed:
updated the description to mention that ALL mercenaries can use it, not just balaamites

Rubeebeebee committed:
Merge branch 'Claymore-branch' of https://github.com/Rubeebeebee/Monolith into Claymore-branch

Redrover1760 reviewed (changes requested):
Seems fine now except

Might be a little underpriced at 186k for its value

Way too much acceleration. Should have a lot less thrusters.

Rubeebeebee commented:
> Seems fine now except
> 
> Might be a little underpriced at 186k for its value
> 
> Way too much acceleration. Should have a lot less thrusters.

i can make it more expensive, no problem

for the acceleration though, thats a good thing! I want this to ram things as a method of delivering balaam's justice! and also because its funny as fuck

Rubeebeebee commented:
> Seems fine now except
> 
> Might be a little underpriced at 186k for its value
> 
> Way too much acceleration. Should have a lot less thrusters.

is it possible to keep the amount of thrusters? or is that many a hard no?

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Claymore-branch

Rubeebeebee committed:
updated the decals, removed some thrusters, upped the price

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> Seems fine now except
> 
> Might be a little underpriced at 186k for its value
> 
> Way too much acceleration. Should have a lot less thrusters.

Right! i have updated the claymore with the requests!

1st, upped the price to 201,100 spessos! The appraisal was around 119k, so an almost a 100k markup! 
<img width="1212" height="230" alt="claymore appraisal" src="https://github.com/user-attachments/assets/d28126a0-db32-461b-8560-e95ef200cf7c" />

2nd, removed some thrusters like requested! Removed 6 thrusters in total 
<img width="976" height="485" alt="claymore less thrusters" src="https://github.com/user-attachments/assets/e3238152-7902-4497-84a4-305b50a58d49" />

3rd, not requested, but ive updated the decal work a little more to fit in the "the most barebones interior for a zealot ship" type of vibe
<img width="921" height="729" alt="deacal tiles rework" src="https://github.com/user-attachments/assets/03b5f572-bb1f-483d-9c76-825e64567daa" />

Redrover1760 reviewed (changes requested):
exterior ship walls should either be reinforced or plastitanium

tiled decals don't tend mesh very well with non-tiled tiles for the wood tiles, maybe try another type of wood.

otherwise I will have to look at it a bit more later

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into Claymore-branch

Rubeebeebee committed:
edited decals for the shrine room, adjusted the thrusters to be closer and use reinforced walls

Rubeebeebee committed:
Merge branch 'Claymore-branch' of https://github.com/Rubeebeebee/Monolith into Claymore-branch

Rubeebeebee commented:
> exterior ship walls should either be reinforced or plastitanium
> 
> tiled decals don't tend mesh very well with non-tiled tiles for the wood tiles, maybe try another type of wood.
> 
> otherwise I will have to look at it a bit more later

fixed the exterior walls by the thrusters to be reinforced walls, and replaced the tiles and decals of the shrine room to hopefully match a little more!
<img width="955" height="265" alt="claymore decals2" src="https://github.com/user-attachments/assets/f5027956-9dab-4373-84ad-3c32083a434f" />
<img width="452" height="358" alt="claymore decals1" src="https://github.com/user-attachments/assets/c6c601d1-39fd-44fa-937c-ce9ee0b5f98e" />

Rubeebeebee commented:
LETS GOOOO!! THANK YOU!!! @Redrover1760
